sdk/include/event.h

```diff
@@ -48,8 +58,22 @@ int __cheri_compartment("sched") event_bits_wait(Timeout  *timeout,
                                                  void     *evt,
                                                  uint32_t *retBits,
                                                  uint32_t  bitsToWait,
-                                                 bool      clearOnExit,
-                                                 bool      waitAll);
+                                                 int       flags __if_cxx(=0)
+                                                 );
+
+
+/**
+ * Flags used by `event_bits_set`.
+ */
+enum [[clang::flag_enum]] EventSetFlags {
+  /** 
+    * Do not yield immediately if a higher priority thread becomes runnable.
+    * Instead the other thread will run when this thread yields or another
+    * scheduler event occurs (e.g. a timer interrupt). This may be useful to
+    * avoid multiple context switches when setting bits in multiple event
+    * groups from a low priority thread.
+    */
+  EventSetNoYield = (1 << 0)};
```

Comment:
I think GitHub has screwed up rendering the diff here, there's no } for this enum definition in the UI.

Reply:
It is there but on the same line as the `)`. `clang-format` seems happy with this.